src/main.js

```diff
@@ -188,10 +188,53 @@ function languageCodeIsCompatible(language, languages) {
     return languages.some(l => l === languageCode);
 }
 
+function flatpakSystemDirs() {
+  let env = GLib.getenv('EOS_DISCOVERY_FEED_FLATPAK_SYSTEM_DIRS');
+  if (env)
+      return env.split(':');
+
+  return ['/var/lib/flatpak', '/var/endless-extra/flatpak'];
+}
+
+// We need to look at the flatpak directories and /var/lib explicitly
```

Comment:
Why `/var/lib`? Nothing installs applications in there.

Reply:
My idea here was to symlink `/usr/share/eos-discovery-feed` to `/var/lib/eos-discovery-feed` so that we could mount it inside the sandbox. But that didn't work. Any ideas?

Reply:
Is this to iterate over the discovery feed provider files? You can give access to `/usr/share/eos-discovery-feed` from the host OS directly through the flatpak manifest, no?

Reply:
Doesn't look like that's possible:
```
sam@endless:~/Source/endless-sdk-flatpak$ flatpak run --command=/bin/bash --filesystem=/usr/share/eos-discovery-feed com.endlessm.apps.Sdk//3
bash-4.3$ ls /usr/share | grep eos-discovery-feed
```

Reply:
I can't think of a great way to do this in the flatpak world; if we don't care about relaxing sandboxing requirements for the feed flatpak, we can run the feed with `--filesystem=host:ro` and access it inside `/run/host/usr` from inside the sandbox.

Reply:
Either way, I don't think we need `/var/lib`

Reply:
> I can't think of a great way to do this in the flatpak world; if we don't care about relaxing sandboxing requirements for the feed flatpak, we can run the feed with --filesystem=host:ro and access it inside /run/host/usr from inside the sandbox

Makes sense. I don't care too much about sandboxing since we had to access the host's /usr/share in some form anyway. I'll fix that now, thanks for the tip.